blueprints/public.py

```diff
@@ -15,7 +19,23 @@ def public_page(page):
 
     return rendered_page
 
-@public.route('/export/')
-def export_event():
-    # The ID of the event will be a GET variable in the query string
-    pass
+@public.route('/export/<eventid>', methods=['POST', 'GET'])
+def export_event(eventid):
+    if request.method == 'GET':
+
+        event_data = db.events.find_one({'_id' : ObjectId(eventid)})
+        cal = Calendar()
+        event = Event()
+        event['dtstart'] = convert_time(str(event_data['start']))
+        event['dtend'] = convert_time(str(event_data['end']))
```

Comment:
Instead of writing your own function, use Python's `datetime` library. Use `datetime.datetime.strptime()`.